src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -18,13 +18,14 @@
 import com.google.api.codegen.config.ApiConfig;
 import com.google.api.codegen.config.InterfaceConfig;
 import com.google.api.codegen.config.MethodConfig;
-import com.google.api.codegen.transformer.ApiMethodTransformer;
+import com.google.api.codegen.transformer.DynamicLangApiMethodTransformer;
```

Comment:
I added a `DynamicLangApiMethodTransformer` for the Ruby change, but it affects Node too. In retrospect, I could've done it in multiple PRs.